src/rasterstats/utils.py

```diff
@@ -25,12 +27,13 @@ def get_percentile(stat):
     return q
 
 
-def rasterize_geom(geom, like, all_touched=False):
+def rasterize_geom(geom, shape, affine, all_touched=False):
```

Comment:
Can we do this without a breaking change to the API? I'd prefer to keep the `like` argument or at least discuss and change this in a separate PR.

Reply:
:+1:  happy to switch this and `rasterize_pctcover_geom` back to using `like` argument before PR is merged.